web/src/typeahead_helper.ts

```diff
@@ -293,11 +305,9 @@ export function compare_people_for_relevance(
     return compare_by_pms(person_a, person_b);
 }
 
-export function sort_people_for_relevance(
-    objs: UserOrMention[],
-    current_stream_id: number,
-    current_topic: string,
-): UserOrMention[] {
+export function sort_people_for_relevance<
```

Comment:
This part got a bit messy -- it's expecting a user or mention (which is possible from the composebox typeahead), but the pill typeahead only has users and I don't want to convert everything in that module to `UserOrMention` because it adds unnecessary complexity. So this function needs to handle both "definitely just a user" and "could be a user or a mention" and _return that same type_.